LayoutTests/fast/repaint/fixed-move-after-keyboard-scroll.html

```diff
@@ -5,7 +5,7 @@
       function runTest()
       {
           if (window.internals)
-              internals.settings.setScrollAnimatorEnabled(false);
+              internals.settings.setScrollAnimatorEnabled(true);
```

Comment:
This test was *specifically* about the ScrollAnimatorEnabled setting, so it seems wrong to be changing this. Maybe disable animated keyboard scrolling in a test header instead?